src/index.js

```diff
@@ -139,7 +139,7 @@ class CategoryLogger extends Logger {
     );
     const minLevel = Logger.levelFromName[levelNameForCategory];
 
-    if (record.level >= minLevel) {
+    if (record.level >= minLevel || record.alwaysEmitLog) {
```

Comment:
This feels like `!important` in CSS: an escape hatch for badly factored codebases. 😔 😩

Reply:
I would like to say that I agree, we need to think through this: https://jupiterone.atlassian.net/browse/DP-1431?focusedCommentId=33796